docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,531 @@
+
+
+```
+# pdn.cfg
+###############################
+# POWER or GROUND #Std. cell rails starting with power or ground rails at the bottom of the core area
+set ::rails_start_with "POWER" ;
+
+# POWER or GROUND #Upper Mal stripes starting with power or ground rails at the left/bottom of the core area
+set ::stripes_start_with "POWER" ;
+
+# Power nets
+set ::power_nets "VDD"
+set ::ground_nets "VSS"
+
+set pdngen::global_connections {
+VDD {
+  {inst_name .* pin_name VPWR}
+  {inst_name .* pin_name VPB}
+}
+VSS {
+  {inst_name .* pin_name VGND}
+  {inst_name .* pin_name VNB}
+}
+}
+#===> Power grid strategy
+# Ensure pitches and offsets will make the stripes fall on track
+pdngen::specify_grid stdcell {
+  name grid
+  rails {
+      M1 {width 1.04 offset 0}
+  }
+  straps {
+      M4 {width 2.08 pitch 20.28 offset 0}
+      M5 {width 2.08 pitch 20.28 offset 0}
+  }
+  connect {{M1 M4} {M4 M5}}
+}
+
+pdngen::specify_grid macro {
+  orient {R0 R180 MX MY}
+  power_pins "VPWR"
+  ground_pins "VGND"
+  blockages "M1 M2 M3 M4"
+  connect {{M4_PIN_ver M5}}
+}
+
+pdngen::specify_grid macro {
+  orient {R90 R270 MXR90 MYR90}
+  power_pins "VPWR"
+  ground_pins "VGND"
+  blockages "M1 M2 M3 M4"
+  connect {{M4_PIN_hor M5}}
+}
+```
+
+
+
+## Tapcell Configuration <a name="TapcellConfiguration"></a>
+
+The tapcell configuration file is used to insert tapcells and endcaps into the design. The tapcell command has the following list of options:
+
+
+```
+    tapcell [-tapcell_master tapcell_master]
+    [-endcap_master endcap_master]
+    [-distance dist]
+    [-halo_width_x halo_x]
+    [-halo_width_y halo_y]
+    [-tap_nwin2_master tap_nwin2_master]
+    [-tap_nwin3_master tap_nwin3_master]
+    [-tap_nwout2_master tap_nwout2_master]
+    [-tap_nwout3_master tap_nwout3_master]
+    [-tap_nwintie_master tap_nwintie_master]
+    [-tap_nwouttie_master tap_nwouttie_master]
+    [-cnrcap_nwin_master cnrcap_nwin_master]
+    [-cnrcap_nwout_master cnrcap_nwout_master]
+    [-incnrcap_nwin_master incnrcap_nwin_master]
+    [-incnrcap_nwout_master incnrcap_nwout_master]
+    [-tap_prefix tap_prefix]
+    [-endcap_prefix endcap_prefix]
+```
+
+
+A simple, yet effective `tapcell.tcl` file is given below:
+
+
+```
+# tapcell.tcl
+###################
+tapcell \
+  -endcap_cpp "2" \
+  -distance 120 \
+  -tapcell_master "FILL1" \
+  -endcap_master "FILL1"
+```
+
+
+
+## setRC Configuration <a name="setRCConfiguration"></a>
+
+
+setRC allows the user to define resistances and capacitances for layers and vias using the `set_layer_rc` command.. This is useful when these values are missing from the tech file. There is also a command that allows you to set the resistance and capacitance of routing wires using the `set_wire_rc`. Following is a generic example of a setRC configuration file which sets the resistance and capacitance of five metal layers, 4 vias, one signal wire, and one clock wire.
```

Comment:
set_wire_rc is required even if it is present in Tech File (Matt Liberty). Remove the implication that this is optional.